packages/notebook/src/browser/contributions/notebook-outline-contribution.ts

```diff
@@ -21,18 +21,16 @@ import { OutlineViewService } from '@theia/outline-view/lib/browser/outline-view
 import { NotebookModel } from '../view-model/notebook-model';
 import { OutlineSymbolInformationNode } from '@theia/outline-view/lib/browser/outline-view-widget';
 import { NotebookEditorWidget } from '../notebook-editor-widget';
-import { NotebookCellModel } from '../view-model/notebook-cell-model';
 import { DisposableCollection, URI } from '@theia/core';
 import { CellKind, CellUri } from '../../common';
 import { NotebookService } from '../service/notebook-service';
 export interface NotebookCellOutlineNode extends OutlineSymbolInformationNode {
-    notebookCell: NotebookCellModel;
-    uri: URI;
+    uri: string;
 }
 
 export namespace NotebookCellOutlineNode {
     export function is(element: object): element is NotebookCellOutlineNode {
-        return TreeNode.is(element) && OutlineSymbolInformationNode.is(element) && 'notebookCell' in element;
+        return TreeNode.is(element) && OutlineSymbolInformationNode.is(element) && 'uri' in element && (element as NotebookCellOutlineNode).uri.startsWith(CellUri.cellUriScheme);
```

Comment:
Suggestion: Since this method is executed on arbitrary `OutlineSymbolInformationNode` instances (many of which use `uri: URI`), this code leads to a lot of errors. You should be more conservative about your type checking here:
```suggestion
        return TreeNode.is(element)
            && OutlineSymbolInformationNode.is(element)
            && isObject<NotebookCellOutlineNode>(element)
            && typeof element.uri === 'string'
            && element.uri.startsWith(CellUri.cellUriScheme);
```